app-editors/scite/scite-4.1.2.ebuild

```diff
@@ -0,0 +1,156 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit flag-o-matic gnome2-utils llvm toolchain-funcs xdg-utils
+
+MY_PV=${PV//./}
+DESCRIPTION="A very powerful, highly configurable, small editor with syntax coloring"
+HOMEPAGE="http://www.scintilla.org/SciTE.html"
+SRC_URI="https://www.scintilla.org/${PN}${MY_PV}.tgz -> ${P}.tgz"
+
+LICENSE="HPND lua? ( MIT )"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86 ~x86-fbsd ~amd64-linux ~arm-linux ~x86-linux"
+IUSE="lua clang"
+
+RDEPEND="dev-libs/glib:=
+	x11-libs/cairo
+	x11-libs/gdk-pixbuf
+	x11-libs/gtk+:3=
+	x11-libs/pango
+	lua? ( >=dev-lang/lua-5:= )
+	clang? ( >=sys-devel/clang-5:= )
+"
+
+DEPEND="${RDEPEND}
+	virtual/pkgconfig"
+
+S="${WORKDIR}/${PN}/gtk"
+
+pkg_pretend() {
+	# use clang could be package specific
+	if use clang && !(tc-is-clang) ; then
```

Comment:
I feel like you don't need this check in here. If you want to go with the route you took, you can **force** clang being used later on. Also, your `clangMajorVersion` is not the correct way to do this, since you can get the version info from llvm.eclass.

Reply:
Sorry, I meant toolchain-funcs.eclass.